### DISCUSSION TITLE
mastermind link

### DISCUSSION THREAD

**Discussion by JMdeSteAgathe** [vidboda/MobiDetails#83] [Ideas]
hi,
it would be helpful to have a mastermind link. indeed, live search can get recent variant that won't be displayed in UCSC mastermind track
https://mastermind.genomenon.com/detail?gene=AIFM1&mutation=NC_000023.11:g.130139809T>C

**Comment by vidboda** [comment]
should be possilbe in a future release

**Comment by vidboda** [comment]
done with commit 8e2843f
